website/docs/failing-closed.md

```diff
@@ -0,0 +1,148 @@
+---
```

Comment:
if this is a new page, we need to add to sidebar for visiblity
https://github.com/open-policy-agent/gatekeeper/blob/master/website/sidebars.js

Reply:
Good find, adding